src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.html

```diff
@@ -402,7 +402,7 @@ <h1 class="gct-heading h2">Gene Comparison Tool</h1>
                           '). You must unpin some genes before you can pin more.'
                         : 'Add all matching genes to Pinned list'
                     "
-                    tooltipPosition="top"
+                    tooltipPosition="left"
```

Comment:
In Firefox, the tooltip would appear on top of the button when the screen was very wide, but not when the screen was narrow. I suspect that the bug was related to how PrimeNG calculates the tooltip position when the tooltip is close to the edge of the screen. Moving the tooltip to the left of the button ensures that the tooltip is shown even when the screen is narrow.

Reply:
Looks great!